### PR TITLE
delay call to get_details

### PR DESCRIPTION
**Pull request by Robert-Costello**
## Technical Summary

temporary fix to account for es update lag.
### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by MartinRiese — APPROVED** [review]

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js [hunk 1]
```diff
@@ -312,8 +312,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
 
             $.when(FormplayerFrontend.getChannel().request("icon:click", currentUrlToObject)).done(function () {
-                self.reloadCase(self.model.get('id'));
-                resetIcon();
+                setTimeout(() => {
+                    self.reloadCase(self.model.get('id'));
+                    resetIcon();
+                }, 500);
```
Comment (orangejenny) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:318]: half a second is enough to account for ES lag?

Comment (Robert-Costello) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:318]: In QA there was a ~50% success rate with getting the accurate case change before calling `get_details`, so I think based on that, this was the estimation of what would/could work. 
@shubham1g5 correct me if I'm missing anything. 
I'll be testing on staging and we could increase it if needed.

Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:318]: Interesting, okay. With this marked as a temporary fix, is there a different long-term plan?

Comment (Robert-Costello) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:318]: My understanding is that we'll need to create a new ticket to investigate further how case search is interacting with clickable icons. The 50% success/failure rate is when case search is used in conjunction with clickable icons.

Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:318]: okay, got it

Comment (shubham1g5) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:318]: That's right. This limit is definitely a rough limit at this moment and is not going to work in all cases for eg. when ES is non performant due to pillow lag or any other infra issues. It's more of a temporary solution to probably try and address 90% of the cases but we definitely expect this to come up as a bug pretty soon.

**Pull request merged**